src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
```

Comment:
The variable names `tempKyesRobot` and `tempValueRobot` contain typos and are not descriptive of their purpose. Consider renaming them to `robotKeys` and `robotValues` respectively for clarity.